.github/images/entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+
+echo -e "\n\n--------- Starting the configuration of rskj ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/rskj.git rskj
+cd rskj && git checkout $RSKJ_BRANCH
+chmod +x ./configure.sh && chmod +x gradlew
+./configure.sh
+./gradlew --no-daemon clean build -x test
```

Comment:
I think there's no need to build the rskj project. The federate-node project will build it automatically while the federate-node is building.

Reply:
Agree with @jeremy-then